src/goto-programs/adjust_float_expressions.cpp

```diff
@@ -20,6 +20,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "goto_model.h"
 
+/// iterate over an expression and check it or any of its subexpressions are
```

Comment:
I'm not sure there are strict rules, but I'd start with a capital letter and also finish off with a full stop. That's to distinguish those brief notes that comments often are from proper documentation (which this should be).